src/commands/init.ts

```diff
@@ -27,10 +29,26 @@ export default class Init extends BaseCommand {
   }]
 
   async run() {
-    const codex = new Codex()
-    await codex.init({
-      specUrl: this.args.url,
-      format: this.flags.format
-    })
+    ui.warning(`bucket ${config.get('s3.bucket')} will be reset`)
+
+    const { Toggle } = require('enquirer')
+
+    const prompt = new Toggle({
+      message: `Are you sure you want to re-init the current bucket (${config.get('s3.bucket')})?`,
```

Comment:
```suggestion
      message: `Are you sure you want to init the current bucket (${config.get('s3.bucket')})?`,
```
If this warning is showing when the bucket is initialized for the first time, I would suggest this error message